common/common.go

```diff
@@ -18,34 +18,49 @@ var (
 	ZeroHash = common.HexToHash("0x0")
 )
 
+const (
+	Uint32ByteSize = 4
+	Uint64ByteSize = 8
+)
+
 // Uint64ToBytes converts a uint64 to a byte slice
 func Uint64ToBytes(num uint64) []byte {
-	const uint64ByteSize = 8
-
-	bytes := make([]byte, uint64ByteSize)
+	bytes := make([]byte, Uint64ByteSize)
 	binary.BigEndian.PutUint64(bytes, num)
 
 	return bytes
 }
 
 // BytesToUint64 converts a byte slice to a uint64
 func BytesToUint64(bytes []byte) uint64 {
```

Comment:
Any reason why we are making changes here?

Reply:
I have noticed [this](https://github.com/agglayer/aggkit/pull/191/files#diff-18eb3102bf480ea624d689abce486ee67aa3ec80ca545c2d7952ff12dfffeea6L391-L393) private function and decided it is better to reuse the one from the `common` pkg. However the code in the `common.go`  needed some changes (to be reused for both converting the byte array with less than 4 bytes and the one that is exactly 4 bytes in length) and better code coverage and that's how I ended up affecting this logic.